lib/rules/color-named/index.js

```diff
@@ -96,7 +79,13 @@ const rule = (primary, secondaryOptions) => {
 				}
 
 				// Check for named colors for "never" option
-				if (primary === 'never' && type === 'word' && namedColors.includes(value.toLowerCase())) {
+				if (
+					primary === 'never' &&
+					type === 'word' &&
+					/^[a-z]+$/iu.test(value) &&
+					value.toLowerCase() !== 'transparent' &&
+					colord(value).isValid()
```

Comment:
[question] Is this `/^[a-z]+$/iu` check (lower-case check) necessary, that is, is it enough to use just `colord(value).isValid()`?

```suggestion
					value.toLowerCase() !== 'transparent' &&
					colord(value).isValid()
```

Reply:
I think we need to check if it is defined in the alphabet to avoid hex (e.g. `#fff`).

Reply:
> `/^[a-z]+$/iu` check (lower-case check)

It is not a lower-case check because it has an `i` flag.

Reply:
Thanks, I see. 👍🏼